CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Changelog
 
+## v1.1.0 (2019-2-28)
+
+### :gear: TypeScript 3 Support ([#3401](https://github.com/DevExpress/testcafe/issues/3401))
+
+TypeScript test files can now use new syntax features introduced in [TypeScript 3.0](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html) and [TypeScript 3.3](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-3.html).
```

Comment:
"use new syntax features introduced" -> use **the** new syntax features introduced